.travis.yml

```diff
@@ -186,7 +186,6 @@ script:
   fi;
 
 after_success:
-- bash <(curl -s https://codecov.io/bash)
```

Comment:
Recommend commenting this out (with a brief explanation of why it's disabled) rather than deleting it outright.

Reply:
done